x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group4/run_soon.ts

```diff
@@ -22,15 +22,15 @@ export default function createRunSoonTests({ getService }: FtrProviderContext) {
     const objectRemover = new ObjectRemover(supertest);
 
     before(async () => {
-      await esArchiver.load('x-pack/test/functional/es_archives/rules_scheduled_task_id');
+      await esArchiver.load('x-pack/test/functional/es_archives/rules_run_soon');
```

Comment:
Created a separate data archive for this test suite because I encountered race conditions when unloading / loading the same archive that was used in the previous test suite. A task would run and be unable to find the rule causing it to delete itself. When in reality we are playing with the underlying SOs by unloading and loading archives at the same time as the task was running.